### ISSUE TITLE
The MNIST training example contains two type errors

### ISSUE THREAD

**Issue by nzmora** [fastai/fastai#2288]
**Describe the bug**

The MNIST training example contains two cut&paste errors:
1.```data = ImageDataBunch.from_folder(path, ds_tfms=tfms, bs=64).normalize(imagenet_stats)```
uses `imagenet_stats` instead of 'mnist_stats.
2. A wrong model is used:
```learn = cnn_learner(data, models.resnet18, metrics=accuracy)```
uses `models.resnet18` which is an ImageNet model.

I would issue a PR, but I couldn't find an MNIST model in fastai/models.

**Provide your installation details**

**To Reproduce**

Run `$ python3 examples/train_mnist.py`
**Expected behavior**

**Screenshots**

**Additional context**

**Comment by sgugger** [comment]
No, the mnist examples uses an imagenet pretrained model, so it's normal to use the `imagenet_stats` on which the model was pretrained. This is an example of use of transfer learning.

**Comment by vincentmathis** [comment]
I think it should be made more clear that all models in fast.ai use the pretained version from torchvision which have all been pretrained on imagenet and should all be fed with images that are normalized with imagenet_stats.

I was thoroughly confused as why there are also mnist_stats and cifar_stats in fast.ai included when all models have to use imagenet_stats.

**Comment by sgugger** [comment]
Not all models are pretrained, only the one created via the convenience function `cnn_learner`. Users also define their own models and might need those `cifar_stats` or `mnist_stats`. There are plenty of examples of this in the course.

**Comment by vincentmathis** [comment]
Oh I see that makes sense.
I followed the tutorial section https://docs.fast.ai/tutorial.data.html

> 
> Let's begin with our sample of the MNIST dataset.
> ```
> mnist = untar_data(URLs.MNIST_TINY)
> tfms = get_transforms(do_flip=False)
> ```
> It's set up with an imagenet structure so we use it to load our training and validation datasets, then label, transform, convert them into ImageDataBunch and finally, normalize them.
> ```
> data = (ImageList.from_folder(mnist)
>         .split_by_folder()          
>         .label_from_folder()
>         .transform(tfms, size=32)
>         .databunch()
>         .normalize(imagenet_stats))
> ```

Which i read as that the MNIST dataset is set up like the ImageNet dataset (concerning folder structure etc.) and wondered why it says `.normalize(imagenet_stats)` when we are using the MNIST dataset.

It isn't clear in the beginners tutorial that the model which is used in the tutorial linked above was pretrained on imagenet. So i somehow ended up thinking I should use mnist_stats for the MNIST dataset.

I guess I should have just read more carefully, thanks for clearing that up.

**Comment by sgugger** [comment]
Note that any PR clarifying the tutorial is welcome ;). If it wasn't clear to you, chances are it's not clear for other people as well.
